### ISSUE TITLE
convert relative filepath for security reason

### ISSUE THREAD

**Issue by Jxck** [power-assert-js/power-assert#20]
power-asserted test result includes `filepath` with 'full path' like this.

```js
assert.strictEqual(assert._expr(assert._capt(typeof res, 'arguments/0'), {
  content: 'assert.strictEqual(typeof name, \'string\')',
  filepath: '/home/username/path/to/project/and/file.js',
  line: 22
}), 'string');
```

file path with **full path** is kind of sensitive information in server or developer machine.
if user doesn't know this behavior, it may cause some security issue.
(for example, some could forgot adding build result to gitignore and push to public repository.)

I think this information is only a project specific.
so it seems better for me to use relative path of test file or project or configurable base bath.

any comments ?

**Comment by twada** [comment]
@Jxck Thank you for your comment. You are absolutely right.

Using relative paths is already planned in next release, but now I'm reminded I have to hurry.
- [Duplication in the error's stack and message](https://github.com/power-assert-js/power-assert/issues/19#issuecomment-96369991)
- [sourceRoot option](https://github.com/power-assert-js/espower/pull/18)

I'm so sorry for my insensitiveness. Next version will be released soon.
Thank you so much.

**Comment by Jxck** [comment]
[:+1:](http://lgtm.in/u/qHTl7iePp) [:-1:](http://lgtm.in/r/qHTl7iePp)

thanks, close this when resolved.

**Comment by Jxck** [comment]
it seems some commits but not fixed yet in current version
any updates or not ?

**Comment by twada** [comment]
@Jxck Sorry for slow progress. power-assert is now in 1.0.0 release process. To be released today or tomorrow.

**Comment by Jxck** [comment]
🙏🍻🍻🍻🙏

**Comment by twada** [comment]
@Jxck, instrumentors below are using `^1.0.0` of `espower` (or `espower-source`) module so they have fixed.

 - [espower-loader](http://github.com/power-assert-js/espower-loader/releases/tag/v1.0.0)
 - [intelli-espower-loader](https://github.com/power-assert-js/intelli-espower-loader/releases/tag/v1.0.0)
 - [espower-cli](http://github.com/power-assert-js/espower-cli/releases/tag/v1.0.0)
 - [espowerify](http://github.com/power-assert-js/espowerify/releases/tag/v1.0.0)
 - [grunt-espower](http://github.com/power-assert-js/grunt-espower/releases/tag/v1.0.0)
 - [gulp-espower](http://github.com/power-assert-js/gulp-espower/releases/tag/v1.0.0)
 - [espower-coffee](http://github.com/power-assert-js/espower-coffee/releases/tag/v1.0.0)
 - [babel-plugin-espower](https://github.com/power-assert-js/babel-plugin-espower/releases/tag/v1.0.0)
 - [webpack-espower-loader](https://github.com/power-assert-js/webpack-espower-loader/releases/tag/v1.0.0).
 - [espower-typescript](http://github.com/power-assert-js/espower-typescript/releases/tag/v1.0.0)

More to come:
 - [espower-babel](https://github.com/power-assert-js/espower-babel)
 - [karma-espower-preprocessor](https://github.com/power-assert-js/karma-espower-preprocessor)
 - [espower-traceur](https://github.com/yosuke-furukawa/espower-traceur)

**Comment by twada** [comment]
This issue is fixed by the [1.0.0 release](https://github.com/power-assert-js/power-assert/releases/tag/v1.0.0) of power-assert family.
Feel free to reopen this if it is not fixed.

**Comment by Jxck** [comment]
great work thanks :)
